doc-content/enterprise-only/getting-started/assets-types-ref.adoc

```diff
@@ -4,12 +4,9 @@ Anything that can be versioned in the {CENTRAL} repository is an asset. A projec
 
 The following image shows the available assets in {PRODUCT} {PRODUCT_VERSION}.
 
-image::getting-started/asset-list.png[]
```

Comment:
I think _asset-list.png_ is no longer needed, could you please remove it?

Reply:
Ack.